### PR TITLE
Spectest Fixes

### PR DESCRIPTION
**Pull request by bjfish**
- Invalid table size for elems should produce link error
- Remove filter which prevents initializer offset check

### CODE REVIEW

File: lib/runtime-core/src/backing.rs [hunk 1]
```diff
@@ -237,9 +237,9 @@ impl LocalBacking {
                     let table = &tables[local_table_index];
 
                     if (table.size() as usize) < init_base + init.elements.len() {
-                        let delta = (init_base + init.elements.len()) - table.size() as usize;
-                        // Grow the table if it's too small.
-                        table.grow(delta as u32).expect("couldn't grow table");
```
Comment (syrusakbary) [inline lib/runtime-core/src/backing.rs]: Just double-checking. We don't want to grow the table in this case, right?

Comment (bjfish) [reply lib/runtime-core/src/backing.rs]: Yes, not to grow appears to be the correct implementation. I checked with one other implementation, the minimum table size should be enough to fit the elems intializer.

**Comment by syrusakbary** [comment]
Two test failed on Windows:
```
clif:fail:data.wast:186 # AssertUnlinkable - caught panic Any
clif:fail:data.wast:194 # AssertUnlinkable - caught panic Any
```

**Pull request merged**